Explorer/Assets/DCL/MapRenderer/MapLayers/PlayerMarker/PlayerMarkerController.cs

```diff
@@ -47,6 +48,9 @@ public void Initialize()
             playerMarker = builder(instantiationParent);

```

Comment:
Why the need of the lazy instantiation? Do you need this complexity? Cant you just instantiated right away?

Reply:
It is to be in line with the other layers as well, the creation of markers is delegated to the installers and then the controller just relies on the installer logic for the marker creation